src/geo/transform.js

```diff
@@ -394,6 +394,17 @@ class Transform {
         return new Float32Array(posMatrix);
     }
 
+    /**
+     * Calculate the distance from the center of a tile to the camera
+     * @param {Tile} tile
+     */
+    cameraToTileDistance(tile: Object) {
+        const posMatrix = this.calculatePosMatrix(tile.coord, tile.sourceMaxZoom);
+        const tileCenter = [tile.tileSize / 2, tile.tileSize / 2, 0, 1];
+        vec4.transformMat4(tileCenter, tileCenter, posMatrix);
+        return tileCenter[3];
+    }
```

Comment:
I hesitate to say "pixels" because that's not quite right, but it's distance in view-space coordinates, which is pixel-derived. It's the same units as "cameraToCenterDistance", which is defined as:

`0.5 / Math.tan(this._fov / 2) * this.height` (and `height` is in pixels)